tests for sync module

Scandie opened the pull request:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/openprocurement/openprocurement.client.python/59)
<!-- Reviewable:end -->

Scandie committed:
add tests for sync and utils modules

VDigitall commented on openprocurement_client/tests/tests_sync.py:
```diff
@@ -0,0 +1,401 @@
+        response = get_response(mock_client, {})
+        log_strings = self.log_capture_string.getvalue().split('\n')
+        self.assertEqual(log_strings[0], 'PreconditionFailed: Not described error yet.')
+        self.assertEqual(log_strings[1], '')
```
Такі рядки можна прибрати, вони не несуть ніякого змісту

Scandie committed:
add test for .feeder() and .get_resource_items methods

Scandie committed:
increase test coverage

set nosetests configurations
fix get_latest_plans() and get_latest_tenders() methods
